src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
I got it. Thinking now, maybe the `validateContext` has a bad name and should be changed sometime in future. Guys (@albertossilva @agutoli), do you have any thoughts ?